### PR TITLE
docs: update grids and charts sections + api docs - PTL-1520 PTL-1522

### PR DESCRIPTION
**Pull request by MrBrunoWolff**
ATENTION! YOU SHOULD NOW BRANCH FROM PREPROD WHEN YOU UPDATE 

  - Please check the [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4) for details
  
__________

Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?

no

Have you done a trial build to check all new or changed links?

Yes

Is there anything else you would like us to know?

TBD

**This week's exciting advice from the style guide**

- Write your headings in sentence case:

  - A good example
    This is a correct heading.
  - A Bad Example
    This is not a correct heading.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,18 +37,21 @@
...
+    "@genesislcap/foundation-forms": "14.214.3",
+    "@genesislcap/foundation-header": "14.214.3",
+    "@genesislcap/foundation-layout": "14.214.3",
+    "@genesislcap/foundation-login": "14.214.3",
+    "@genesislcap/foundation-reporting": "14.214.3",
+    "@genesislcap/foundation-testing": "14.214.3",
+    "@genesislcap/foundation-zero": "14.214.3",
+    "@genesislcap/g2plot-chart": "14.214.3",
+    "@genesislcap/grid-pro": "14.214.3",
+    "@genesislcap/grid-tabulator": "14.214.3",
```
Comment (matteematt) [inline package.json:54]: Nitpick: bumping all the packge versions imply that they've also had their docs pages sycned, is that the case? If not I think we should leave the older package versions as they were and bump and sync them in a different PR.

File: plugins/api-docs/dist/manifest.js [hunk 2]
```diff
@@ -284,5 +262,114 @@ exports.default = {
...
+                api_docs: "docs/api",
+                readme: "01_grid-pro.mdx",
+                keywords: ["genesis", "foundation", "ui", "grid", "grid-pro", "ag"],
+                tags: [
+                    "connected",
+                    "datasource",
+                    "grid",
+                    "grid-pro",
+                    "ag",
+                ],
```
Comment (matteematt) [inline plugins/api-docs/dist/manifest.js:306]: Doesn't necessarily need to be done in this PR but I am aware some feedback we've been getting from the docs is that it's difficult to search for things. I am thinking due to that we should try add as much relevent bits to these as possible to try and aid searchability. Won't block the merge for this for now, but equally if you can think of more ideas then that would be ideal. Same with the other new two blocks you've added

File: plugins/api-docs/processedMap.js [hunk 3]
```diff
@@ -8,13 +8,16 @@ module.exports = {
...
+	"@genesislcap/foundation-filters": "14.214.3",
+	"@genesislcap/foundation-forms": "14.214.3",
+	"@genesislcap/foundation-header": "14.214.3",
+	"@genesislcap/foundation-layout": "14.214.3",
+	"@genesislcap/foundation-login": "14.214.3",
+	"@genesislcap/foundation-reporting": "14.214.3",
+	"@genesislcap/foundation-testing": "14.214.3",
+  "@genesislcap/g2plot-chart": "14.214.3",
+  "@genesislcap/grid-pro": "14.214.3",
+  "@genesislcap/grid-tabulator": "14.214.3",
```
Comment (matteematt) [inline plugins/api-docs/processedMap.js:22]: Tiny thing, but the indentation here doesn't match. Also see my other point about the other package versions

File: plugins/api-docs/src/manifest.ts
Comment (matteematt) [inline plugins/api-docs/src/manifest.ts:1]: Just realised I commented the tags and keywords message on the js file rather than this 😅

**Review by matteematt — COMMENTED** [review]
Really great work, thanks a bunch for picking this up! Left a few comments but I think they're fairly minor. However a larger point - double check with Jay but I think we should be targetting the `new-docs-structure` branch with changes for now

File: plugins/api-docs/processedMap.js (hunk 3 above)
Comment (MrBrunoWolff) [reply plugins/api-docs/processedMap.js:22]: will check

File: plugins/api-docs/src/manifest.ts
Comment (MrBrunoWolff) [reply plugins/api-docs/src/manifest.ts:1]: ahh got it, makes sense now

**Comment by matteematt** [comment]
> > Really great work, thanks a bunch for picking this up! Left a few comments but I think they're fairly minor. However a larger point - double check with Jay but I think we should be targetting the `new-docs-structure` branch with changes for now
> 
> yah, wasn't aware of that branch when this was added
> 
> will try to switch it + check the new branch

BTW Jay sent me a typo in the branch name, the new branch name is `new-doc-structure`

**Comment by MrBrunoWolff** [comment]
closing this in favor of https://github.com/genesiscommunitysuccess/docs/pull/1857 (targetting new-doc-structure branch)

**Pull request closed without merging**